### ISSUE TITLE
What is needed to support partners overcall? (#80)

### ISSUE THREAD

**Issue by ThorvaldAagaard** [EdwardPiwowar/BBA#52]
404025F24F21D5F8BF6F991B52DF
![image](https://github.com/EdwardPiwowar/BBA/assets/10560652/1278ab72-d62f-449d-a0fe-6dc32101c7bc)

with 7 HCP and 3-card support it is ok to support a 1-level overcall

**Comment by EdwardPiwowar** [comment]
But it's very ugly 7.

**Comment by ThorvaldAagaard** [comment]
I agree, but partner likes getting supported. After partners 1S the hand gets a lot better, so selling out to 3C also looks very bad
